src/components/Reports/ReportsFilters/ReportsFilters.js

```diff
@@ -50,9 +49,7 @@ const ReportsFilters = ({
                             <FormattedDate value={domainSelected.verified_date} />
                           </span>
                         </>
-                      ) : (
-                        <FormattedMessageMarkdown id="reports_filters.domain_not_verified_MD" />
-                      )}
+                      ) : null}
```

Comment:
Are you sure a null value here is ok?

Reply:
because the message now show before the boxes